ZeldaFullEditor/Gui/GfxGroupsForm.cs

```diff
@@ -487,7 +487,13 @@ private void createPalette()
                             pPos++;
                         }
 
-                        palettes[pPos] = Palettes.OverworldAuxPalettes[GfxGroups.paletteGfx[(int)paletteUpDown.Value][0]][i];
+                        int value = GfxGroups.paletteGfx[(int)paletteUpDown.Value][0];
+                        if (value >= 20)
```

Comment:
Let's make this and the other instances of 20 and 19 into constansts which explain what the upper bound signifies

Reply:
Done